docs/documentation/getting-started/millionaires.md

```diff
@@ -122,8 +164,10 @@ The output being `0` tells Elon that unfortunately Jeff is still the alpha:
 
 ```shell
 [0]
-    creation-date -> 1630661192626
-    gameID -> 7899b23c-4509-4ff8-a9ae-d9b59fa77fea
+	gameID -> 56effe1e-d85c-4af0-8894-846f3b62e4db
```

Comment:
Is it intended that the indention has changed here?

Reply:
no, replaced tabs with spaces